.rubocop_todo.yml

```diff
@@ -0,0 +1,326 @@
+# This configuration was generated by `rubocop --auto-gen-config`
+# on 2015-04-27 23:58:52 +0200 using RuboCop version 0.30.1.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
```

Comment:
Oops. Sorry, I made a small mistake. Not just `git add .rubocop.yml`, but `git add --patch .rubocop.yml` and then add the newly generated stuff and don't remove what was manually added to / changed in the file earlier. In other words, selectively add new stuff to the config file. That's what I ususally do. I also use [SourceTree](http://www.sourcetreeapp.com) for picking chunks or even individual lines for commits.